code/game/gamemodes/nations/nations.dm

```diff
@@ -72,6 +73,14 @@ datum/game_mode/nations
 				H << "You are now part of the great sovereign nation of [H.mind.nation.name]!"
 				continue
 
+			if(H.mind.assigned_role in servicion)
+				H.mind.nation = all_nations["Servicion"]
+				H.hud_updateflag |= 1 << NATIONS_HUD
```

Comment:
While I do get that this is unanimous for all of hud code... this is still so fucking bad. Why does it use the shift operator in combination with the |= operator!? I DON'T UNDERSTAND THIS!